app/models/course/assessment/question/programming_test_case.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 class Course::Assessment::Question::ProgrammingTestCase < ActiveRecord::Base
+  acts_as_duplicable
```

Comment:
It's not useful. Put it in for consistency and an explicit declaration that it must have the `initialize_duplicate` function.

I'll remove it then.